lmnet/tests/lmnet_tests/test_visualize.py

```diff
@@ -70,8 +71,20 @@ def test_semantic_segmentation():
     assert not np.all(np.array(input_image) == np.array(result_image))
 
 
+def test_keypoint_detection():
+    """Verify just image is changed."""
+    input_image = PIL.Image.new("RGB", size=(100, 200))
+    joints = np.zeros(shape=(17, 3), dtype=np.int)
+    joints[0] = [30, 30, 1]
+
+    result_image = visualize_keypoint_detection(np.array(input_image), joints)
+
+    assert not np.all(np.array(input_image) == np.array(result_image))
```

Comment:
is this the same as 
`assert np.all(... != ...)`?

Reply:
No, they are different.
`assert not np.all(np.array(input_image) == np.array(result_image))` means some elements changed.
`assert np.all(... != ...)` means all elements changed.

Reply:
ahh then `np.any(... != ...)` ?